galata/test/jupyterlab/menus.test.ts

```diff
@@ -176,3 +176,180 @@ function reduceItem(
   }
   return null;
 }
+
+test.describe('Top menu keyboard navigation', () => {
+  test.use({ autoGoto: false });
+  test('navigate to open file with keyboard', async ({ page }) => {
+    await page.goto();
+    for (let i = 0; i < 3; i++) {
+      await page.keyboard.press('Shift+Tab');
```

Comment:
Why are we using shift+tab rather than tab?

Reply:
Thanks, have amended to tab